tests/cggi/straight_line_vectorizer.mlir

```diff
@@ -1,5 +1,5 @@
 // TODO(#519): disable FileChecks until nondeterminism issues are resolved
-// RUN: heir-opt --straight-line-vectorize %s
+// RUN: heir-opt --straight-line-vectorize %s | FileCheck %s
```

Comment:
For this test: use this gist: https://gist.github.com/asraa/81aaa9dea83a3e67620420c4439b864a

There are some non-determinism issues (linked as a TODO in this test above), but there are only two cases for it, and we can at least check that the `cggi.lut3` was vectorized there.